src/lib/utils/evm/index.ts

```diff
@@ -44,6 +44,7 @@ type BuildAndBroadcastArgs = BuildArgs &
 
 type CreateBuildCustomTxInputArgs = {
   accountNumber: number
+  pubKey: string
```

Comment:
nitpick - now looking at the old `Api` functions, we use `from` vs `pubKey` which I feel makes more sense in the evm context. it is basically the same as the api funcs except for `wallet` instead of `supportsEIP1559`. would be my pref as long as we are doing this.

Reply:
Absolutely aligned on that - much cleaner with consistent vernacular. b8d45026b2a11ddc4f64d12d673e2cee7a0ee9d9